### PR TITLE
docs(gen): content review

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: containers/kubernetes/how-to/use-scratch-storage-h100.mdx [hunk 1]
```diff
@@ -14,26 +14,26 @@ categories:
...
-- **Experimentation and output evaluation:** Experimenting with different configurations and hyperparameters often involves iterative processes. H100 scratch storage makes quick storage and retrieval of intermediary results easy, enabling efficient experimentation and evaluation of model outputs.
+- **Temporary storage for preprocessing:** For data preprocessing tasks that demand high I/O throughput and low latency, H100 and L40S scratch storage offers a temporary space on the host node's filesystem. This can be advantageous for quick and efficient preprocessing steps in complex data pipelines.
+- **Experimentation and output evaluation:** Experimenting with different configurations and hyperparameters often involves iterative processes. H100 and L40S scratch storage makes quick storage and retrieval of intermediary results easy, enabling efficient experimentation and evaluation of model outputs.
```
Comment (jcirinosclwy) [inline containers/kubernetes/how-to/use-scratch-storage-h100.mdx:24]: I'm wondering if it should be singular here: `H100 and L40S scratch storage **make** quick storage and retrieval...`
@RoRoJ

File: containers/kubernetes/how-to/use-scratch-storage-h100.mdx [hunk 2]
```diff
@@ -14,26 +14,26 @@ categories:
...
 
 - A Scaleway account logged into the [console](https://console.scaleway.com)
 - [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
-- [Created](/containers/kubernetes/how-to/create-cluster) a Kubernetes Kapsule or Kosmos cluster that uses [H100 GPU Instances](https://www.scaleway.com/en/h100-pcie-try-it-now/)
+- [Created](/containers/kubernetes/how-to/create-cluster) a Kubernetes Kapsule or Kosmos cluster that uses [H100 and L40S GPU Instances](https://www.scaleway.com/en/h100-pcie-try-it-now/)
 
-In Kubernetes, H100 scratch storage is implemented as a directory on the host node's filesystem. This scratch volume is made available to containers within a Pod using the `hostPath` volume type, allowing direct integration with Kubernetes workloads.
+In Kubernetes, H100 and L40S scratch storage is implemented as a directory on the host node's filesystem. This scratch volume is made available to containers within a Pod using the `hostPath` volume type, allowing direct integration with Kubernetes workloads.
```
Comment (jcirinosclwy) [inline containers/kubernetes/how-to/use-scratch-storage-h100.mdx:34]:
```suggestion
In Kubernetes, H100 and L40S scratch storage is implemented as a directory on the host node's filesystem. This scratch volume is made available to containers within a pod using the `hostPath` volume type, allowing direct integration with Kubernetes workloads.
```

File: containers/kubernetes/how-to/use-scratch-storage-h100.mdx [hunk 3]
```diff
@@ -14,26 +14,26 @@ categories:
...
 
-Configuring H100 scratch storage in Kubernetes involves specifying the `hostPath` volume type in the Pod's volume definition. The following is an example configuration:
+Configuring H100 and L40S scratch storage in Kubernetes involves specifying the `hostPath` volume type in the Pod's volume definition. The following is an example configuration:
```
Comment (jcirinosclwy) [inline containers/kubernetes/how-to/use-scratch-storage-h100.mdx:36]:
```suggestion
Configuring H100 and L40S scratch storage in Kubernetes involves specifying the `hostPath` volume type in the pod's volume definition. The following is an example configuration:
```

File: tutorials/load-overflow-proxmox-and-elastic-metal/index.mdx [hunk 4]
```diff
@@ -1,18 +1,18 @@
 ---
 meta:
-  title: Dealing with load overflow between a Proxmox cluster and virtual machines on an Elastic Metal Server over Scaleway Private Networks
-  description:  This tutorial shows how to deal with load overflow between a Proxmox cluster and virtual machines on an Elastic Metal Server over Scaleway Private Networks
+  title: Managing load overflow - Proxmox cluster and VMs on Scaleway Elastic Metal
+  description:  Learn to manage load overflow between a Proxmox cluster and VMs on an Elastic Metal Server via Scaleway Private Networks. Step-by-step tutorial.
```
Comment (jcirinosclwy) [inline tutorials/load-overflow-proxmox-and-elastic-metal/index.mdx:4]:
```suggestion
  description: Learn to manage load overflow between a Proxmox cluster and VMs on an Elastic Metal Server via Scaleway Private Networks. Step-by-step tutorial.
```

**Review by jcirinosclwy — APPROVED** [review]

**Review by nerda-codes — APPROVED** [review]

**Pull request merged**